prometheus_hardware_exporter/collector.py

```diff
@@ -531,7 +531,7 @@ def fetch(self) -> List[Payload]:
         """Load ipmi sel entries."""
         sel_entries = self.ipmi_sel.get_sel_entries(self.config.ipmi_sel_interval)
 
-        if not sel_entries:
+        if sel_entries is None:
             logger.warning("No recent ipmi sel entries to collect.")
```

Comment:
Since this happens only in case of error now, I think we can change the log message
```suggestion
            logger.warning("Failed to get ipmi sel entries.")
```